src/subcommand/init.rs

```diff
@@ -215,6 +226,38 @@ _z_cd() {{
 use posix_z as bash_z;
 use posix_z as zsh_z;
 
+fn powershell_z(z_cmd: &str) -> String {
+    format!(
+        r#"
+function {} {{
+    function z_cd($dir) {{
+        try {{
+            Set-Location $dir -ea Stop
+            if ($env:_ZO_ECHO -eq "1") {{
+                Write-Host "$PWD"
+            }}
+        }} catch {{
+        }}
+    }}
+
+    if ($args.Length -eq 0) {{
+        z_cd ~
+    }}
+    elseif ($args.Length -eq 1 -and $args[0] -eq '-') {{
+        z_cd -
+    }}
+    else {{
+        $result = zoxide query @args
+        if ($LASTEXITCODE -eq 0) {{
```

Comment:
`zoxide query --help` also returns an exit code of `0`, but we wouldn't want to `cd` in that case. Could you change this to `cd` only when `$result` is a valid directory, and, in the `else` clause, print out `$result`?

Reply:
@ajeetdsouza - Fixed